boards/sodaq-sara-aff/Makefile

```diff
@@ -1,8 +1,5 @@
 MODULE = board
 
-ifneq (,$(filter boards_common_samd21-arduino-bootloader,$(USEMODULE)))
-  # Add the samd21-arduino-bootloader directory to the build
-  DIRS += $(RIOTBOARD)/common/samd21-arduino-bootloader
-endif
+DIRS += $(RIOTBOARD)/common/sodaq
```

Comment:
Are you sure of this change (I can't test) ? Because like this you are not building the `board.c` file of the sodaq-sara-aff but the common one which is in `boards/common/sodaq`.
Is the reboot in bootloader still working ?

Reply:
Well, I don't have the AFF board, but I'm doing the very same thing with the SFF board.
In both cases it compiles two `board.c` files and it creates these two relevant .a files:
- `board.a` (with the board specific `board.o`)
- `boards_common_sodaq.a` (with the common `board.o`)
The link command names `board.a` first, so I'm guessing that it will pick up the code from `boards/sodaq-sara/sff/board.c`.
I hope it was designed to work this way, because now you can fall back to the common `board.c` if you don't have one in the normal boards directory.

Anyway, I've tested this with SFF by changing `board.c` to switch on `LED_RED`. And I see that the red led goes on after flashing.

Reply:
Next, what do you mean by "Is the reboot in bootloader still working?"

Reply:
Ok, I understand. The "reset in bootloader" is built from the common board module. But it seems dangerous to me to build both modules (the sodaq common and the in the board). I would prefer to leave it as it was before and apply the same for sodaq-sara-sff.

Reply:
OK, I removed the AFF change, and I changed the SFF Makefile to be like sodaq-sara-aff, and sodaq-one.
Note, I took the liberty to already rebase and squash